server/apps/postgres-app/src/test/java/org/apache/james/PostgresWithTikaTest.java

```diff
@@ -0,0 +1,44 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james;
+
+import static org.apache.james.data.UsersRepositoryModuleChooser.Implementation.DEFAULT;
+
+import org.apache.james.backends.postgres.PostgresExtension;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class PostgresWithTikaTest implements JamesServerConcreteContract {
+    static PostgresExtension postgresExtension = PostgresExtension.empty();
+
+    @RegisterExtension
+    static JamesServerExtension jamesServerExtension = new JamesServerBuilder<PostgresJamesConfiguration>(tmpDir ->
+        PostgresJamesConfiguration.builder()
+            .workingDirectory(tmpDir)
+            .configurationFromClasspath()
+            .searchConfiguration(SearchConfiguration.openSearch())
+            .usersRepository(DEFAULT)
+            .build())
+        .server(PostgresJamesServerMain::createServer)
+        .extension(new DockerOpenSearchExtension())
+        .extension(new TikaExtension())
```

Comment:
Q: Will we support tika in Postgres?

(just re-confirm)

Reply:
Some injections with the search module chooser required the tika module. So I had to test it. But if not tika properties file, it is disabled and we use jsoupextractor instead. From what I understand when checking quickly